pyam/core.py

```diff
@@ -1404,12 +1409,21 @@ def check_aggregate_region(
             (currently only supported with `method='sum'`)
         exclude_on_fail : boolean, optional
             flag scenarios failing validation as `exclude: True`
+        drop_negative : bool, default True
```

Comment:
Explicitly writing the default in the docstring is not in line with numpy-doc convention, better to simply write optional (see [here](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).

The pyam code doesn't consistently follow this convention, but we should try to apply the proper format in any new code (and clean up older functions as we go along).
```suggestion
        drop_negative : bool, optional
```